docs/features/software-catalog/api.md

```diff
@@ -456,6 +460,35 @@ where the `items` array has _the same length_ and _the same order_ as the input
 `entityRefs` array. Each element contains the corresponding entity data, or
 `null` if no entity existed in the catalog with that ref.
 
+### `POST /refresh`
+
+Refresh the entity related to `entityRef`.
+
+Request body is JSON, on the form
+
+```json
+{
+  "entityRef": "<string>",
+  "authorizationToken": "<string>"
+}
+```
+
+### `POST /validate-entity`
+
+Validate that a passed in entity has no errors in schema.
+
+Request body is JSON, on the form
+
+```json
+{
+  "location": "<string>",
+  "entity": {
+    "proident_f": {},
+    "culpa_ca": {}
```

Comment:
Are these intended to be examples? Something doesn't look right here.

Reply:
Corrected